certified-connectors/Ordnance Survey Places/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2985 @@
+                          },
+                          "BLPU_STATE_CODE_DESCRIPTION": {
+                            "type": "string",
+                            "description": "BLPU_STATE_CODE_DESCRIPTION"
+                          },
+                          "TOPOGRAPHY_LAYER_TOID": {
+                            "type": "string",
+                            "description": "TOPOGRAPHY_LAYER_TOID"
+                          },
+                          "LAST_UPDATE_DATE": {
+                            "type": "string",
+                            "description": "LAST_UPDATE_DATE"
+                          },
+                          "ENTRY_DATE": {
+                            "type": "string",
+                            "description": "ENTRY_DATE"
+                          },
+                          "STREET_STATE_CODE": {
+                            "type": "string",
+                            "description": "STREET_STATE_CODE"
+                          },
+                          "STREET_STATE_CODE_DESCRIPTION": {
+                            "type": "string",
+                            "description": "STREET_STATE_CODE_DESCRIPTION"
+                          },
+                          "STREET_CLASSIFICATION_CODE": {
+                            "type": "string",
+                            "description": "STREET_CLASSIFICATION_CODE"
+                          },
+                          "STREET_CLASSIFICATION_CODE_DESCRIPTION": {
+                            "type": "string",
+                            "description": "STREET_CLASSIFICATION_CODE_DESCRIPTION"
+                          },
+                          "LPI_LOGICAL_STATUS_CODE": {
+                            "type": "string",
+                            "description": "LPI_LOGICAL_STATUS_CODE"
+                          },
+                          "LPI_LOGICAL_STATUS_CODE_DESCRIPTION": {
+                            "type": "string",
+                            "description": "LPI_LOGICAL_STATUS_CODE_DESCRIPTION"
+                          },
+                          "LANGUAGE": {
+                            "type": "string",
+                            "description": "LANGUAGE"
+                          },
+                          "MATCH": {
+                            "type": "number",
+                            "format": "float",
+                            "description": "MATCH"
+                          },
+                          "MATCH_DESCRIPTION": {
+                            "type": "string",
+                            "description": "MATCH_DESCRIPTION"
+                          }
+                        },
+                        "description": "LPI"
+                      }
+                    }
+                  },
+                  "description": "results"
+                }
+              }
+            }
+          }
+        },
+        "summary": "OS Places API Find Operation",

```

Comment:
@quickjc1 This summary value can be improved, something like below:

`Find Operation`

Applies to all the similar summary values in the swagger

Reply:
@sriyen-msft agreed, will change